### PR TITLE
LCR - Assignment-1

### PR DESCRIPTION
**Pull request by troniak**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Evaluate a KNN model to classify wine based on chemical properties

## What did you learn from the changes you have made?
The optimal parameter for n_neighbors using 10-fold cross validation was 15

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I thought about calculating accuracy by hand instead of using accuracy_score

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Had to refer to the documentation on train_test_split

## How were these changes tested?
Using the wine dataset in sklearn

## Checklist
- [ x] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by KasraVakiloroayaei — CHANGES_REQUESTED** [review]
A1: Incomplete 

Q2part(ii) since it is a categorical variable, we do not standardize it 
Q3: The parameter grid is missing or incorrectly defined. It appears that you have the wrong identified the wrong value for your hyperparameter k, nearest Neighbors 
Q4 - Please calculate the accuracy once again after making changes finding a new best n-neighbor

**Comment by troniak** [comment]
> Q3: The parameter grid is missing or incorrectly defined. It appears that you have the wrong identified the wrong value for your hyperparameter k, nearest Neighbors

I don't think there is a problem with the parameter grid. I added stratify to the train test split to get a best n-neighbor of 7

**Review by KasraVakiloroayaei — APPROVED** [review]
A1: Complete 

The changes you have made look good. Thanks
